colossalai/inference/tensor_parallel/kvcache_manager.py

```diff
@@ -0,0 +1,102 @@
+# Adapted from lightllm/common/mem_manager.py
+# of the ModelTC/lightllm GitHub repository
+# https://github.com/ModelTC/lightllm/blob/050af3ce65edca617e2f30ec2479397d5bb248c9/lightllm/common/mem_manager.py
+
+import torch
+
+from colossalai.logging import get_dist_logger
```

Comment:
This relies on gpc, which will be deprecated soon. Please don't use it.

Reply:
remove it!